Sources/SpeziAccount/Model/AdditionalRecordId.swift

```diff
@@ -11,7 +11,7 @@
 ///
 /// The identifier is built by combining a stable ``AccountService`` identifier and the primary accountID (see ``AccountIdKey``).
 /// Using both, additional data records of a user can be uniquely identified across ``AccountService`` implementations.
-public struct AdditionalRecordId: CustomStringConvertible, Hashable, Identifiable {
+public struct AdditionalRecordId: CustomStringConvertible, Hashable, Identifiable, Sendable {
```

Comment:
Afaik, structs that only contain `Sendable` types should automatically conform to `Sendable`, see here: https://www.hackingwithswift.com/swift/5.5/sendable
However, I guess the `Hashable` or `Identifiable` implementation will require the manual `Sendable` conformance.